src/ripple/core/impl/SociDB.cpp

```diff
@@ -196,44 +197,33 @@ namespace {
     is the default behavior of sqlite. We may be able to remove this
     class.
 */
+
 class WALCheckpointer : public Checkpointer
 {
 public:
-    WALCheckpointer(sqlite_api::sqlite3& conn, JobQueue& q, Logs& logs)
-        : conn_(conn), jobQueue_(q), j_(logs.journal("WALCheckpointer"))
+    WALCheckpointer(
+        std::uint64_t id,
```

Comment:
Should be `std::uintptr_t`

Reply:
Agreed. Fixed in `1a499c7b4 Styles fixes and use correct int type to cast to pointer`